BangazonAPITest/TestControllers/PaymentTypeControllerTest.cs

```diff
@@ -0,0 +1,257 @@
+
+            }
+
+
+            [Fact]
+
+            public async Task Delete_PaymentType()
+            {
+            // Note: with many of these methods, I'm creating dummy data and then testing to see if I can delete it. I'd rather do that for now than delete something else I (or a user) created in the database, but it's not essential-- we could test deleting anything 
+
+            // Create a new coffee in the db
+            PaymentType newBitCash = await CreateDummyPaymentType();
+
+                // Delete it
+                await deleteDummyPaymentType(newBitCash);
+
+                using (var client = new APIClientProvider().Client)
+                {
+                    // Try to get it again
+                    HttpResponseMessage response = await client.GetAsync($"{url}{newBitCash.Id}");
+
+                    // Make sure it's really gone
+                    Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
+
+                }
+            }
+
+            [Fact]
+            public async Task Get_All_PaymentType()
+            {
+
+                using (var client = new APIClientProvider().Client)
+                {
+
+                // Try to get all of the PaymentTypes from /api/PaymentType
+                HttpResponseMessage response = await client.GetAsync(url);
+                    response.EnsureSuccessStatusCode();
+
+                    // Convert to JSON
+                    string responseBody = await response.Content.ReadAsStringAsync();
+
+                    // Convert from JSON to C#
+                    List<PaymentType> paymentTypes = JsonConvert.DeserializeObject<List<PaymentType>>(responseBody);
+
+                // Make sure we got back a 200 OK Status and that there are more than 0 PaymentTypes in our database
+                Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+                    Assert.True(paymentTypes.Count > 0);
+
+                }
+            }
+
+            [Fact]
+            public async Task Get_Single_PaymentType()
+            {
+                using (HttpClient client = new APIClientProvider().Client)
+                {
+                // Create a dummy coffee
+                PaymentType newBitCash = await CreateDummyPaymentType();
+
+                    // Try to get it
+                    HttpResponseMessage response = await client.GetAsync($"{url}/{newBitCash.Id}");
+                    response.EnsureSuccessStatusCode();
+
+                    // Turn the response into JSON
+                    string responseBody = await response.Content.ReadAsStringAsync();
+
+                // Turn the JSON into C#
+                PaymentType bitCashFromDB = JsonConvert.DeserializeObject<PaymentType>(responseBody);
+
+                    // Did we get back what we expected to get back? 
+                    Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+                    Assert.Equal(dummyPaymentType.AcctNumber, newBitCash.AcctNumber);
```

Comment:
```suggestion
                    Assert.Equal(dummyPaymentType.AcctNumber, newBitCash.AcctNumber);
```
I believe this may be producing a false positive by not comparing to something in the database